tests/system/system_tests.cfg

```diff
@@ -1,395 +1,451 @@
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#VP_ITER=VP_ITER_VP_ITER_CONVERGE
+#
+#[System-run_with_vp_interp_never]
+#driver = classic 
+#test_description = complete simulation if MTCLIM_SWE_CORR=VP_ITER_NEVER
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#VP_ITER=VP_ITER_NEVER
+#
+#[System-run_with_vp_interp]
+#driver = classic 
+#test_description = complete simulation if VP_INTERP=TRUE
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#VP_INTERP=TRUE
+#
+#[System-run_without_vp_interp]
+#driver = classic
+#test_description = complete simulation if VP_INTERP=FALSE
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#VP_INTERP=FALSE
+#
+#[System-run_with_lw_cloud_deardroff]
+#driver = classic 
+#test_description = complete simulation if LW_CLOUD=LW_CLOUD_DEARDORFF
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#LW_CLOUD=LW_CLOUD_DEARDORFF
+#
+#[System-run_with_lw_cloud_bras]
+#driver = classic 
+#test_description = complete simulation if LW_CLOUD=LW_CLOUD_BRAS
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#LW_CLOUD=LW_CLOUD_BRAS
+#
+#[System-fail_if_carbon_without_veglibphoto]
+#driver = classic 
+#test_description = fail simulation if CARBON=TRUE without VEGLIB_PHOTO
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 1
+#check = None
+#[[options]]
+#CARBON=TRUE
+#VEGLIB_PHOTO=FALSE
+#
+#[System-fail_if_veglib_photo_without_veglib]
+#driver = classic 
+#test_description = fail simulation if VEGLIB_PHOTO=TRUE without input data
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 1
+#check = None
+#[[options]]
+#VEGLIB_PHOTO=TRUE
+#
+#[System-run_with_rc_mode_jarvis]
+#driver = classic 
+#test_description = complete simulation if RC_MODE=RC_JARVIS
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#RC_MODE=RC_JARVIS
+#
+#[System-run_with_rc_mode_photo]
+#driver = classic
+#test_description = complete simulation if RC_MODE=RC_PHOTO
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#RC_MODE=RC_PHOTO
+#
+#[System-run_baseflow_arno]
+#driver = classic
+#test_description = complete simulation if BASEFLOW=ARNO
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#BASEFLOW=ARNO
+#
+#[System-run_baseflow_nijssen]
+#driver = classic
+#test_description = complete simulation if BASEFLOW=NIJSSEN2001
+#global_parameter_file = global_param.ROSEM.system.txt
+#expected_retval = 0
+#check = None
+#[[options]]
+#BASEFLOW=NIJSSEN2001
+#'''
+
+[System-check-domainfile-and-nans] 
+driver = image
```

Comment:
This is more like an issue for later discussion, not necessarily for this PR I guess - so based on our conversation yesterday, I agree with you that it would be cleaner to put the driver option here for each individual system test. But currently driver as well as VIC executable file are still input arguments to `run_tests.py`. I feel this is a little confusing with the driver specification here, but I guess for unit tests we do want to specify the driver from input command line (right?). So maybe we can talk how we want to handle this eventually...